pom.xml

```diff
@@ -52,13 +52,12 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins.workflow</groupId>
             <artifactId>workflow-api</artifactId>
-            <version>2.2-beta-2</version> <!-- allows consuming the new APIs -->
+            <version>2.2</version> <!-- allows consuming the new APIs -->
         </dependency>
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>pipeline-input-step</artifactId>
             <version>2.0</version>
-            <optional>true</optional>
```

Comment:
We're tied to the 2.0 APIs now, this is to ensure they're pulled in (potential fix for some dependency issues too).